boards/BMS/Src/Io/Io_LTC6813/Io_SharedLTC6813.c

```diff
@@ -182,7 +190,7 @@ static void Io_PrepareCfgRegBytes(
     // cell location
     uint8_t min_cell_segment = 0U;
     uint8_t min_cell_index   = 0U;
-    Io_NewCellVoltages_GetMinCellLocation(&min_cell_segment, &min_cell_index);
+    Io_CellVoltages_GetMinCellLocation(&min_cell_segment, &min_cell_index);
 
     // Write to the last segment on the daisy chain first
```

Comment:
Does the last segment on the daisy chain correspond to segment 0?

Reply:
yeah i think i see where your confusion is coming from

this is rlly stupid i think i have a better way to do this. stay tuned
![image](https://user-images.githubusercontent.com/32230980/158760377-07bbb72e-e3dc-49ab-ba99-b36d1732c93b.png)

Reply:
implementation should be much nicer now